src/genophenocorr/analysis/predicate/_api.py

```diff
@@ -135,3 +135,14 @@ def get_categories() -> typing.Sequence[PatientCategory]:
         The predicate bins a patient into :class:`BooleanPredicate.NO` or :class:`BooleanPredicate.YES` category.
         """
         return BooleanPredicate.NO, BooleanPredicate.YES
+    
+class GroupingPredicate(PolyPredicate, metaclass=abc.ABCMeta):
+    FIRST = PatientCategory(1, 'First', 'The patient belongs in the first group.')
```

Comment:
I would like to start the `id`s from `0` and not from `1`. This will not have any impact on the functionality now, but it would be better to be consistent.

Please start from `0`.